test/TensorFlowNET.UnitTest/layers_test/flatten.cs

```diff
@@ -45,5 +45,14 @@ public void Case4()
             var input = tf.placeholder(TF_DataType.TF_INT32, new TensorShape(3, 4, None, 1, 2));
             sess.run(tf.layers.flatten(input), (input, np.arange(3 * 4 * 3 * 1 * 2).reshape(3, 4, 3, 1, 2))).Should().BeShaped(3, 24);
         }
+
+        [TestMethod]
+        public void Case5()
+        {
+            var sess = tf.Session().as_default();
+
+            var input = tf.placeholder(TF_DataType.TF_INT32, new TensorShape(None, 4, 3, 1, 2));
```

Comment:
Can we change to `var input = tf.placeholder(tf.int32, (None, 4, 3, 1, 2));`

Reply:
You'll have to add 6-12 overloads to support all tuples, feel free to add it yourself.